src/main/java/at/hannibal2/skyhanni/features/nether/KuudraAPI.kt

```diff
@@ -0,0 +1,75 @@
+package at.hannibal2.skyhanni.features.nether
+
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.events.KuudraCompleteEvent
+import at.hannibal2.skyhanni.events.KuudraEnterEvent
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.events.LorenzWorldChangeEvent
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class KuudraAPI {
+
+    private val group = RepoPattern.group("data.kuudra")
+    private val scoreboardGroup = group.group("scoreboard")
+    private val chatGroup = group.group("chat")
+
+    private val tierPattern by scoreboardGroup.pattern(
+        "tier",
+        " §7⏣ §cKuudra's Hollow §8\\(T(?<tier>.*)\\)"
+    )
+
+    private val completePattern by chatGroup.pattern(
+        "complete",
+        "§.\\s*(?:§.)*KUUDRA DOWN!"
+    )
```

Comment:
```suggestion
    private val patternGroup = RepoPattern.group("data.kuudra")

    private val tierPattern by patternGroup.pattern(
        "scoreboard.tier",
        " §7⏣ §cKuudra's Hollow §8\\(T(?<tier>.*)\\)"
    )
    private val completePattern by patternGroup.pattern(
        "chat.complete",
        "§.\\s*(?:§.)*KUUDRA DOWN!"
    )
```
Seeing as there is only two patterns may as well do something like this. Also this is made on web so no clue if it compiles though it should